commons-core/src/main/java/com/fincity/saas/commons/core/enums/ConnectionType.java

```diff
@@ -13,7 +13,17 @@ public enum ConnectionType {
 
     TEXT_MESSAGE,
 
-    REST_API(ConnectionSubType.REST_API_BASIC, ConnectionSubType.REST_API_AUTH, ConnectionSubType.REST_API_OAUTH2);
+    REST_API(ConnectionSubType.REST_API_BASIC, ConnectionSubType.REST_API_AUTH, ConnectionSubType.REST_API_OAUTH2),
+
+    NOTIFICATION(
+            ConnectionSubType.NOTIFICATION_DISABLED,
+            ConnectionSubType.NOTIFICATION_EMAIL_SMTP,
+            ConnectionSubType.NOTIFICATION_EMAIL_SENDGRID,
+            ConnectionSubType.NOTIFICATION_IN_APP,
+            ConnectionSubType.NOTIFICATION_MOBILE_PUSH,
+            ConnectionSubType.NOTIFICATION_WEB_PUSH,
+            ConnectionSubType.NOTIFICATION_SMS);
+    ;
```

Comment:
Notification will not have subtypes but in the connection details we shall take the name of the connect that to be used in each channel.